packages/cli/src/constructs/construct.ts

```diff
@@ -18,7 +22,7 @@ export abstract class Construct {
     return false
   }
 
-  abstract synthesize(): any
+  abstract synthesize(): any|null
```

Comment:
@umutuzgur , I think we should export the new synthesized types/interfaces in each constructs and import them here to indicate the synthesize() return types using `|`.